go/ai/prompt.go

```diff
@@ -426,3 +430,149 @@ func renderDotprompt(templateText string, variables map[string]any, defaultInput
 	}
 	return str, nil
 }
+
+// LoadPromptFolder loads prompts and partials from the input directory for the given namespace
+func LoadPromptFolder(registry *registry.Registry, dir string, ns string) {
+	if dir == "" {
+		dir = "./prompts"
+	}
+	promptsPath, err := filepath.Abs(dir)
+	if err != nil {
+		log.Printf("Failed to resolve directory: %v", err)
```

Comment:
Let's return an error if the prompt dir specified doesn't exist then only log errors if the prompts can't be parsed. The directory might be tough to specify correctly so you'd want to know if it's wrong but then you might have some broken .prompt files in there that you don't care about so you don't want to necessarily error on init for those.